pymodbus/framer/rtu_framer.py

```diff
@@ -127,7 +119,7 @@ def resetFrame(self):
         _logger.debug("Resetting frame - Current Frame in "
                       "buffer - {}".format(hexlify_packets(self._buffer)))
         self._buffer = b''
-        self._header = {}
+        self._header = {'uid': 0x00, 'len': 0, 'crc': b'\x00\x00'}
```

Comment:
This change makes `self._header` content consistent with other framer classes, and resolves issues when some functions expect that this dict always contain some keys.